node/service/src/chain_spec/moonbase.rs

```diff
@@ -224,7 +224,7 @@ pub fn testnet_genesis(
 			// Pretty hacky. We just set the first staker to use alice's session keys.
 			// Maybe this is the moment we should finally make the `--alice` flags make sense.
 			// Which is to say, we should prefund the alice account. Actually, I think we already do that...
-			author_ids: stakers
+			mappings: stakers
```

Comment:
Does it require change in the spec file too ?

Reply:
This change will be necessary in all spec files that are expected to work with the new runtime. That does not include networks that were launched before this PR and then upgraded to include it.

The CI is green which I _think_ means everything in this repo that needs updated was updated.